packages/sanity/src/core/bundles/components/panes/BundleActions.tsx

```diff
@@ -67,7 +71,7 @@ export function BundleActions(props: BundleActionsProps): ReactNode {
     // set up the listener before executing
     const createVersionSuccess = firstValueFrom(
       documentStore.pair
-        .operationEvents(versionId, documentType)
+        .operationEvents(getPublishedId(documentId), documentType)
```

Comment:
Hey @RitaDias why do we need this change?

Reply:
I think I might have left the explanation on the previous PR but it's on the description: fixed issue when adding a version to release made the code that came after the createVersionSuccess not run (wrong id was being passed)

Basically the "success" message never came back because technically it never succeeded (the createVersionSuccess check). This is a separate thing from the actual creating of the version but it listens to the success based on the publishedId. You can see [here](https://github.com/sanity-io/sanity/blob/corel-149-2/packages/sanity/src/core/store/_legacy/document/document-store.ts#L177-L180) :)

Reply:
Got it! Thanks !